<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesDialog</class>
 <widget class="QDialog" name="PropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="cmdNewProperty">
           <property name="text">
            <string>New Property</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdSaveChanges">
           <property name="text">
            <string>Save Changes</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdDeleteProperty">
           <property name="text">
            <string>Delete Property</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="trvProperties">
         <column>
          <property name="text">
           <string>Properties</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="txtCode"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Code:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="txtPropertyName"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Property Name</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Street:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Company:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="cboCompany"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>No. Of Uits:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="txtNoOfUnits"/>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="txtLocation"/>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QLineEdit" name="txtStreet"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>L/R No.</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QLineEdit" name="txtLRNo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Units:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cmdAddUnit">
           <property name="text">
            <string>Add Unit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdRemoveUnit">
           <property name="text">
            <string>Remove Unit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="trvUnits">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <column>
      <property name="text">
       <string>Unit No.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>No. Of Rooms</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sq. Ft.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Occupied</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tenant Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rent Per Month</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmdNewProperty</tabstop>
  <tabstop>cmdSaveChanges</tabstop>
  <tabstop>cmdDeleteProperty</tabstop>
  <tabstop>trvProperties</tabstop>
  <tabstop>cboCompany</tabstop>
  <tabstop>txtPropertyName</tabstop>
  <tabstop>txtCode</tabstop>
  <tabstop>txtLocation</tabstop>
  <tabstop>txtStreet</tabstop>
  <tabstop>txtNoOfUnits</tabstop>
  <tabstop>txtLRNo</tabstop>
  <tabstop>cmdAddUnit</tabstop>
  <tabstop>cmdRemoveUnit</tabstop>
  <tabstop>trvUnits</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
